### DISCUSSION TITLE
keyboard lagging

### DISCUSSION THREAD

**Discussion by CortexQuack** [prasanthrangan/hyprdots#726] [Q&A]
Hi, my usb keyboard keeps lagging, when i do journalctl --follow, it shows that the usb device keeps resetting when it starts to lag, i got an nvidia gpu and razer keyboard, tried another keyboard but got the same result, im not getting this when im in the tty

- what i mean by lag is like it either stops working or types the same key until i press something else like thiiiiiiis

- please tell me if i should ve mentioned something else, im kind of a noob when it comes to linux -_-

**Comment by kRHYME7** [comment]
Is it not your hardware failing? like the hardrive failing ?

**Reply by CortexQuack** [reply]
not too sure, it works perfectly fine on windows and nixos, i also tried another keyboard and it was the same

**Reply by kRHYME7** [reply]
I see, You're using arch? try to use your keyboard without Hyprland to test. ``` CTRL ALT 3 ``` For tty 3. This will open a session In terminal outside Hyprland and try to test something here.

**Reply by CortexQuack** [reply]
i kind of deleted arch after that, but i did try using the tty and didnt get the keyboard lag, i also mentioned that in the discussion title here

**Reply by kRHYME7** [reply]
Yow, I have this weird bug at this moment, Happens only on hyprland session as you also experiences.

**Reply by CortexQuack** [reply]
> Yow, I have this weird bug at this moment, Happens only on hyprland session as you also experiences.

damn, good luck trying to find a solution, please post it here if you do

**Reply by kRHYME7** [reply]
Ow can we at least know your device specs? 

https://github.com/prasanthrangan/hyprdots/discussions/116

**Reply by CortexQuack** [reply]
> Ow can we at least know your device specs? 

(edit, sorry, didnt see the post)
 CPU : i5 gen 1
 GPU : NVIDIA GTX 750 ti
 GPU Driver : yeah no clue, ig the ones that the script installed
 Memory : 6 gb
- and a very old motherboard, think 2013?(it might be because of the motherboard but idk, havent really checked that....)
- keyboard: razer cynosa

tell me if you need any other info about the specs

**Reply by kRHYME7** [reply]
I see, this issue occurred after I turned on my Nvidia dGPU so maybe NVIDIA, these are assumptions though, I will try to log here for every time this problem occurs. Also might as well search for related issues. It just feels good I'm not only the one having this. haha

**Reply by CortexQuack** [reply]
> I see, this issue occurred after I turned on my Nvidia dGPU so maybe NVIDIA, these are assumptions though, I will try to log here for every time this problem occurs. Also might as well search for related issues. It just feels good I'm not only the one having this. haha

so did you find a solution or?

**Reply by kRHYME7** [reply]
Luckily, I did not get this issue again. Have you tried only using Hyprland without configs?.
